specification.md

```diff
@@ -151,6 +156,16 @@ A `String` representing the [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/I
 US
 ```
 
+### CurrencyCode
+
+A `String` representing the [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) code of the given currency.
+
+#### Example
+
+```json
+CAD
```

Comment:
It's not a string unless it has double quotes in JSON :wink:

Reply:
Good eye 👀

Fixed here and in another spot; can you tell that I copy-pasted from that other code block? 🙃